app/views/results/marker/_marker_panes.html.erb

```diff
@@ -14,7 +14,7 @@
               <li><a href='#feedback_file_tab'><%= t('marker.feedback_files') %></a></li>
           <% end %>
 
-           <% if can_show_remark_request_tab_in_marker_pane(submission) %>
+           <% if !@current_user.student? && can_show_remark_request_tab_in_marker_pane(submission) %>
```

Comment:
Change this in the definition of `can_show_remark_request_tab_in_marker_pane` directly.